crates/rustc_codegen_spirv/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/EmbarkStudios/rust-gpu"
 
 [lib]
 crate-type = ["dylib"]
+test = false
```

Comment:
what does this do? there are a few tests in `rustc_codegen_spirv/linker/test.rs`, does this disable those?

Reply:
I... this is an accident, sorry! I was trying to debug reducing my RLS waiting time (it didn't work *sigh*) and accidentally included it in a commit apparently (wouldn't have happened if I was still using `git gui` oh well).